### PR TITLE
Docker compose setup

### PR DESCRIPTION
**Pull request by sfc-gh-plasek**
Adds docker compose setup with tools for backend etc.

Also has non-working profile for running the application under docker-compose, some fixes necessary in that area.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    - type: bind
+      source: ./docker/pgpasswords
+      target: /pgpasswords
+  application:
+    profiles:
+      - all
+    depends_on:
+      database-postgres:
+        condition: service_started
+    container_name: pomagamy
```
Comment (sfc-gh-krzymkowski) [inline docker-compose.yml:55]: How about an english name?

File: docker/environment.env [hunk 2]
```diff
@@ -0,0 +1,8 @@
+PGADMIN_DEFAULT_EMAIL=admin@admin.pl
```
Comment (sfc-gh-krzymkowski) [inline docker/environment.env:1]: Maybe some COI email?

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
 spring.datasource.driver-class-name=org.postgresql.Driver
-spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
-spring.datasource.username=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/ogloszenia
```
Comment (sfc-gh-krzymkowski) [inline src/main/resources/application.properties:2]: Shouldn't we be using eng names? Like `ads` or z`helpoffers`

**Review by sfc-gh-krzymkowski — APPROVED** [review]

File: docker/environment.env (hunk 2 above)
Comment (sfc-gh-plasek) [reply docker/environment.env:1]: If someone proposes one, I have no idea what to put in as it's just used inside the container and nowhere else

File: src/main/resources/application.properties (hunk 3 above)
Comment (sfc-gh-plasek) [reply src/main/resources/application.properties:2]: will do

**Pull request closed without merging**